src/ontology/edits/oeo-social.omn

```diff
@@ -357,7 +357,7 @@ Class: OEO_00000238
 Class: OEO_00000315
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An organization role is a role of a society that has a collective goal and a set of organization rules.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An organization role is a role of a population that has a collective goal and a set of organization rules.",
```

Comment:
That makes no sense what so ever.  A population are the *all* people in a certain region.  This way, members of an organisation have to all live inside an area, and nobody who's not a member of the organisation can live there.  Talk about ~ethnic~ organisational cleansing.

Reply:
yeah right I forgot the spatial region part. So new idea to the society/population def problem: 
Define an "aggregate of people" superclass which is used in the organization def and make population a subclass?

Reply:
if we need a society we can still add it later as another subclass

Reply:
I now just changed the organization role def to "aggregate of people"

Reply:
> classes should arise from concepts rather than from words

What would be the concept relating "society" and "population"?

Reply:
An Aggregate of people